CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # CHANGELOG
 
+### [v2.4 _(Jan 11, 2018)_](https://github.com/omise/omise-magento/releases/tag/v2.5)
```

Comment:
@jacstn 2.5 pls

Reply:
@jacstn 2.5 pls

Reply:
because I changed it in my code already... 😆